search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -22,7 +22,7 @@ function onClickCountry(e) {
   // Update Modal with information for relevant country.
   const countryName = e.feature.getProperty('name');
   const countryId = e.feature.getId();
-  const countryData = e.feature.getProperty('country_data').toLocaleString();
+  //const countryData= e.feature.getProperty('country_data').toLocaleString();
```

Comment:
intentional? If so, add a todo/comment. If not needed now and will not be needed, delete.

Reply:
Deleting it